BAZProjectC1/Billboard/View/BillboardViewController.swift

```diff
@@ -40,16 +37,25 @@ class BillboardViewController : UIViewController {
     }
 }
 
-extension BillboardViewController : UICollectionViewDelegate, UICollectionViewDataSource {
+extension BillboardViewController : UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
     
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
         return lstMovies.count
     }
     
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let myCell = collectionView.dequeueReusableCell(withReuseIdentifier: "MovieCell", for: indexPath) as? MovieCell
-        myCell?.configureCell(with: lstMovies[indexPath.row])
+        myCell?.configureCellUpdate(with: lstMovies[indexPath.row])
         return myCell ?? UICollectionViewCell()
     }
     
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
+        
+        let flowayout = collectionViewLayout as? UICollectionViewFlowLayout
+                let space: CGFloat = (flowayout?.minimumInteritemSpacing ?? 0.0) + (flowayout?.sectionInset.left ?? 0.0) + (flowayout?.sectionInset.right ?? 0.0)
+                let size:CGFloat = (collectionView.frame.size.width - space) / 2.0
+                return CGSize(width: size, height: size)
```

Comment:
Cuida la indentación:

Xcode te puede ayudar con eso presionando `control + i`

```suggestion
        let flowayout = collectionViewLayout as? UICollectionViewFlowLayout
        let space: CGFloat = (flowayout?.minimumInteritemSpacing ?? 0.0) + (flowayout?.sectionInset.left ?? 0.0) + (flowayout?.sectionInset.right ?? 0.0)
        let size:CGFloat = (collectionView.frame.size.width - space) / 2.0
        return CGSize(width: size, height: size)
```